tests/test_pipeline.py

```diff
@@ -150,6 +150,14 @@ def test_non_existing(self):
         ret = main(["pipeline", "show", "non-existing"])
         self.assertNotEqual(ret, 0)
 
+    def test_print_locked_stages(self):
+        self.dvc.add("foo")
+        self.dvc.add("bar")
+        self.dvc.lock_stage("foo.dvc")
+
+        ret = main(["pipeline", "show", "foo.dvc", "--locked"])
```

Comment:
Sorry, our pipeline tests are quite lazy :slightly_smiling_face:  Please use something like in `tests/test_metrics.py` as an example to make sure that the command outputs everything as expected. E.g. https://github.com/iterative/dvc/blob/0.35.7/tests/test_metrics.py#L227 .